stackstate_checks_base/stackstate_checks/base/utils/persistent_state.py

```diff
@@ -0,0 +1,140 @@
+# (C) StackState 2020
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import json
+from schematics.models import Model
```

Comment:
this line should look like this `from schematics import Model` as `Model` is imported in `schematics.__init__py`